dev-docs/bidders/ras.md

```diff
@@ -3,7 +3,7 @@ layout: bidder
 title: RingierAxelSpringer
 description: Prebid RingierAxelSpringer Bidder Adapter
 biddercode: ras
-media_types: banner
+media_types: banner, native
```

Comment:
Can you add 

- multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid

if you support multiple formats

See https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter

Reply:
Hello,
this is the link to the second pr

https://github.com/prebid/Prebid.js/pull/11180